frontend/server/src/Controllers/User.php

```diff
@@ -4328,6 +4328,14 @@ public static function isNonUserIdentity(
      * @omegaup-request-param string $third_party_login
      */
     public static function getLoginDetailsForTypeScript(\OmegaUp\Request $r) {
+        if (
+            !is_null(
+                \OmegaUp\Controllers\Session::getCurrentSession()['identity']
+            )
+        ) {
+            header('Location: /');
+            die();
```

Comment:
```suggestion
            throw new \OmegaUp\Exceptions\ExitException();
```

we don't use `die()` because that causes problems for tests.